sherpa/models/tests/test_model.py

```diff
@@ -541,6 +542,24 @@ def test_composite_iter():
     assert parts[3] is out.m2
     assert len(parts) == 4
 
+    assert m(4) == 10
+
+
+def test_composite_iter_ufunc():
+    """Repeat the test, but call a ufunc instead of using the "+" symbol"""
```

Comment:
Which test is being repeated? Over time the tests may get moved around and so it's not necessarily obvious if it's the previous test or not.

Reply:
[I say this because I have written plenty of comments like this and then it has come back to haunt me ;-]